packages/core-services/src/index.ts

```diff
@@ -115,6 +115,7 @@ export {
 
 // TODO think in a way to not have to pass the service name to proxify here as well
 export const Authorization = proxifyWithWait<IAuthorization>('authorization');
+export const Apps = proxifyWithWait<IAppsEngineService>('apps-engine');
```

Comment:
worries me a little that the best way to avoid cyclical imports is to inject the object after